Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/EdgeBrowserContentLocator.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Threading;
+using System.Windows;
+using System.Windows.Automation;
+using JetBrains.Annotations;
+using OpenQA.Selenium;
+using Remotion.Web.Development.WebTesting.Utilities;
+
+namespace Remotion.Web.Development.WebTesting.ScreenshotCreation.BrowserContentLocators
+{
+  public class EdgeBrowserContentLocator : IBrowserContentLocator

```

Comment:
Missing class docs

Reply:
Actually, the other content locators don't use inheritdoc and specify a new doc here.

Reply:
Thanks, fixed.